game/hud/src/widgets/RefillAmmo/index.tsx

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Because this doesn't use redux state management, move this under the _components_ directory.